frb/figures/utils.py

```diff
@@ -46,9 +46,8 @@ def set_bokeh_fontsize(p, fsz):
     Adjust font size for Bokeh axes
```

Comment:
Remove the `fits` files in GALFIT unless they are necessary.  Unclear to me.
@SunilSimha ?